### DISCUSSION TITLE
There is no data earlier than 18-October-2021 in the Vendor API

### DISCUSSION THREAD

**Discussion by paulamascot** [saleweaver/python-amazon-sp-api#491] [Q&A]
Hi, 

Currently I'm accessing the Orders data from Amazon Vendor through the Selling Partner API. We suppose to have available data since January 2020, but through the API,  I could only find available data from October 2021. Is there any limitation in the API to access these historical data? When I call the data for those specific date (from January 2020 to October 2021) the API returns a blank dictionary. 

Best wishes,

Paula

**Comment by timmyscj** [comment]
Amazon recently announced that they are introducing new reports with a 3 year look back window. Although it appears that none of the new reports are working lol

https://developer-docs.amazon.com/sp-api/changelog/march-2022-sp-api-release-announcement
